### PR TITLE
Broadcasts

### PR DESCRIPTION
**Pull request by nehaparmar99**
## Issue that this pull request solves
All Broadcasts page and route added

 Closes: #288 

### Brief description of what is fixed or changed
Added a all broadcasts page, and setup a route for it.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

This is the button added on the broadcasts page

Screenshot (285)

This is the all broadcasts page-> similar to "Get Involved" page and shows ellipses when the content exceeds 500 words  

Screenshot (286)

### CODE REVIEW

File: frontend/src/components/broadcast/allbroadcasts.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          <h1 className="carousel-head">Broadcasts</h1>
+          <div className="dash"></div>
+        </div>
+      </div>
+      <div id="allCards">
+        {arrayCards.map((element, i) => {
+          return (
+            <div className="card-item" key={i}>
+              <a
+                href="https://github.com/HITK-TECH-Community/Community-Website"
```
Comment (Kajol-Kumari) [inline frontend/src/components/broadcast/allbroadcasts.js:56]: please remove these two lines

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@nehaparmar99 please do the following:
1. Fix the `All Broadcasts` button style as it's style is not smooth. Take reference from `admin?` button maybe
2. Please format the code properly
3. Make a new folder under `broadcast` with the name of `all_broadcasts` and put all the `all_broadcasts.js` and `all_broadcasts.css` file in there.

File: frontend/src/components/broadcast/allbroadcasts.js (hunk 1 above)
Comment (nehaparmar99) [reply frontend/src/components/broadcast/allbroadcasts.js:56]: Should the cards then open a modal showing all the content?

**Comment by nehaparmar99** [comment]
@Kajol-Kumari  please review it, I made the requested changes

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari please review it, I made the requested changes

@nehaparmar99 can you please let me know under which program you have made this PR?

**Review by Kajol-Kumari — APPROVED** [review]
LGMT :+1:

**Comment by nehaparmar99** [comment]
> > @Kajol-Kumari please review it, I made the requested changes
> 
> @nehaparmar99 can you please let me know under which program you have made this PR?

Both DWOC and SWOC

**Pull request merged**